cli/determined_cli/agent.py

```diff
@@ -22,19 +22,27 @@ def list_agents(args: argparse.Namespace) -> None:
     r = api.get(args.master, "agents")
 
     agents = r.json()
-    headers = ["Agent Name", "Registered Time", "Slots", "Containers", "Label"]
-    values = [
-        [
-            local_id(agent_id),
-            render.format_time(agent["registered_time"]),
-            len(agent["slots"]),
-            agent["num_containers"],
-            agent["label"],
-        ]
+    agents = [
+        OrderedDict(
+            [
+                ("id", local_id(agent_id)),
+                ("registered_time", render.format_time(agent["registered_time"])),
```

Comment:
Note that we format the timestamp for printing here, which is arguably wrong -- more correct would be to use the ISO8601 format in the JSON output and `format_time` only for the non-JSON case. But that is a bit annoying to implement -- what do you think? @dzhu

Reply:
Yeah, I think that's okay. Depending on how long this code lives, perhaps the JSON/CSV/table selection logic could eventually all be pushed into `tabulate_or_csv` (presumably under a different name at that point :stuck_out_tongue:) and then that could be done properly in one place.